pkg/kubelet/cm/util/cdi/cdi.go

```diff
@@ -39,8 +39,8 @@ const (
 	annotationPrefix = "cdi.k8s.io/"
 )
 
-// generate container annotations using CDI UpdateAnnotations API.
-func generateCDIAnnotations(
+// GenerateCDIAnnotations generate container annotations using CDI UpdateAnnotations API.
+func GenerateCDIAnnotations(
```

Comment:
minor/nonblocking: if we are creating a package called `cdi`, we can rename `GenerateAnnotations` so it reads
`cdi.GenerateAnnotations` which is nicer.

Reply:
+1

Reply:
I have updated this.